### PR TITLE
[12.0] shopinvader_delivery_carrier : Add description field

### PR DESCRIPTION
**Pull request by hparfr**
It's also added by base_delivery_carrier (not in dependencies).

### CODE REVIEW

**Review by PierrickBrun — APPROVED** [review]

File: shopinvader_delivery_carrier/models/delivery_carrier.py [hunk 1]
```diff
@@ -2,12 +2,15 @@
...
 from contextlib import contextmanager
 
-from odoo import api, models
+from odoo import api, fields, models
 
 
 class DeliveryCarrier(models.Model):
     _inherit = "delivery.carrier"
 
+    description = fields.Text()
```
Comment (rousseldenis) [inline shopinvader_delivery_carrier/models/delivery_carrier.py:11]: Wouldn't be typical for a simple module delivery_carrier_description ? Simple to maintain and can give Odoo idea for a simple enhancement for further version :-)

Comment (hparfr) [reply shopinvader_delivery_carrier/models/delivery_carrier.py:11]: A module with only a text field ?

Comment (simahawk) [inline shopinvader_delivery_carrier/models/delivery_carrier.py:11]: I guess you want to show this on the frontend: what about translations?

Comment (simahawk) [reply shopinvader_delivery_carrier/models/delivery_carrier.py:11]: @rousseldenis ?

Comment (rousseldenis) [reply shopinvader_delivery_carrier/models/delivery_carrier.py:11]: That exists in multiple repositories. But I can admit it's a little excessive.

**Comment by hparfr** [comment]
To be closed when this PR is merged https://github.com/OCA/delivery-carrier/pull/283

**Comment by sebastienbeau** [comment]
I have fixed the foward port and included the description here : https://github.com/shopinvader/odoo-shopinvader/pull/739

I close this PR

**Pull request closed without merging**